### ISSUE TITLE
Discuss the format of Autoware's design documents

### ISSUE THREAD

**Issue by kenji-miyake** [autowarefoundation/autoware-documentation#9]

**Comment by xmfcx** [comment]
I agree, It'd be a lot more cleaner this way. and we could directly track the changes based on the source text of the documentation.

But documentation should have versioning.

**Comment by kenji-miyake** [comment]
The result of https://gitlab.com/autowarefoundation/autoware.auto/AutowareAuto/-/issues/1421 was added by https://github.com/autowarefoundation/autoware-documentation/pull/48.
https://autowarefoundation.github.io/autoware-documentation/main/design/component-interfaces/planning-control-vehicle-interface/

We have to keep discussing this.
